includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -4434,6 +4435,28 @@ private function upe_needs_redirection( $payment_methods ) {
 		return 1 === count( $payment_methods ) && 'card' !== $payment_methods[0];
 	}
 
+
+	/**
+	 * Disable WooPay.
+	 *
+	 * This function disables WooPay by updating the option to 'no'
+	 * once the 'disable_woopay' filter is called.
+	 *
+	 * @return boolean False, indicating WooPay is disabled.
```

Comment:
```suggestion
	 * @return boolean False
```

`false` will always returned, whether WooPay has been disabled or not.

Is `return false` needed? Can the function just return `void`?